buildSrc/src/main/groovy/org/opensearch/gradle/test/TestWithSslPlugin.java

```diff
@@ -49,6 +49,7 @@
 public class TestWithSslPlugin implements Plugin<Project> {
 
     @Override
+    @SuppressWarnings({ "unchecked" })
```

Comment:
Why this? Feels unrelated, move into a separate PR.

Reply:
This should fix the warning 
`Note: /Users/pgodithi/git/OpenSearch/buildSrc/src/main/groovy/org/opensearch/gradle/test/TestWithSslPlugin.java uses unchecked or unsafe operations.`
I will add it to new PR later.